src/getAppSyncConfig.js

```diff
@@ -3,10 +3,16 @@ import {
 } from 'amplify-appsync-simulator';
 import { invoke } from 'amplify-util-mock/lib/utils/lambda/invoke';
 import fs from 'fs';
-import { find } from 'lodash';
+import { find, get } from 'lodash';
 import path from 'path';
 
 export default function getAppSyncConfig(context, appSyncConfig) {
+  const cfg = appSyncConfig;
+  // Flattening params
+  cfg.mappingTemplates = cfg.mappingTemplates.flat();
+  cfg.functionConfigurations = cfg.functionConfigurations.flat();
```

Comment:
@marcoreni I am testing this and getting an error on this line
`TypeError: Cannot read property 'flat' of undefined`

That is because I do not have any `functionConfigurations`
We need to default to `[]` all these configs

Reply:
Totally right!